daemon/api.go

```diff
@@ -2591,9 +2591,12 @@ func runSnapctl(c *Command, r *http.Request, user *auth.UserState) Response {
 	if err != nil {
 		return Forbidden("cannot get remote user: %s", err)
 	}
-	// we only allow "get" from regular users in snapctl
-	if uid != 0 && snapctlOptions.Args[0] != "get" {
-		return Forbidden("cannot use %q with uid %d, try with sudo", snapctlOptions.Args[0], uid)
+	// we only allow "get" and -h/--help from regular users in snapctl
+	if uid != 0 {
+		arg := snapctlOptions.Args[0]
+		if arg != "get" && arg != "-h" && arg != "--help" {
```

Comment:
We also need to allow getting help for individual subcommands.

Reply:
Ah, indeed, this will need different approach then, thank you.